fe/fe-core/src/main/java/com/starrocks/sql/analyzer/UpdateAnalyzer.java

```diff
@@ -82,24 +82,22 @@ public static void analyze(UpdateStmt updateStmt, ConnectContext session) {
             }
         }
 
-        if (table.isOlapTable()) {
-            if (session.getSessionVariable().getPartialUpdateMode().equals("column")) {
```

Comment:
why remove this?

Reply:
To support shared-data and shared-nothing mode both.